x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -105,11 +116,11 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
           </EuiText>
         </EuiButtonEmpty>
         <EuiTitle>
-          <h2>{ISOLATE_HOST}</h2>
+          {isolateAction === 'isolateHost' ? <h2>{ISOLATE_HOST}</h2> : <h2>{UNISOLATE_HOST}</h2>}
```

Comment:
maybe: avoid defining the `<h2>` twice by putting the conditional inside of it and around the label value:

```suggestion
          <h2>{isolateAction === 'isolateHost' ?  ISOLATE_HOST : UNISOLATE_HOST}</h2>}
```

Reply:
i think i initially had this but it was giving me some sort or error, but i'll give it another try